src/app/login/models/feedback.model.ts

```diff
@@ -0,0 +1,17 @@
+export class Feedback {
+  public subject: string;
+  public body: string;
+  public email: string;
+  public context: FeedbackContext;
+
+  constructor(subject: string, body: string, email: string, context: FeedbackContext) {
```

Comment:
not sure if we need a constructor here and if this has to be a `class`. It seems to me that we only want to check the shape of the object which we are using in `sendFeedback` so I would use an `interface` here